magento-research/pwa-studio
# Updates GitHub Pages to redirect to pwastudio.io

## Description

This PR replaces the `index` and `404` pages with redirects to [pwastudio.io](https://pwastudio.io).
Proper redirect code taken directly from [WCAG recommendations](https://www.w3.org/TR/WCAG20-TECHS/H76.html).

We should publish these changes right before we move the repository, ensure they work, and then move it.

## Related Issue





Closes #1492 , created when Columbus sailed the ocean blue 🌊 .

## Verification Steps


In a private repo, @jcalcaben is ensuring that modifying these files with redirects does what we expect it to.

We could test these changes and then change them back before we move the repo if we want to ensure they work.

Any commit to this `gh-pages` branch will update the GitHub Pages site.

## Screenshots / Screen Captures (if appropriate)

## Proposed Labels for Change Type/Package




## Checklist:



## Note / Risk

In an effort to not index this old stale content we remove Google Tag Manager code. 
The downside of this is that we won't be able to track when / how many folks are still hitting the old site.
